internal/api/stats/request/routes.go

```diff
@@ -0,0 +1,15 @@
+package request
+
+// All the API endpoints handled by this package.
+
+import (
+	"github.com/edulinq/autograder/internal/api/core"
+)
+
+var routes []core.Route = []core.Route{
+	core.MustNewAPIRoute(`stats/request/query`, HandleQuery),
```

Comment:
I don't like the name "request" for this endpoint.
It is way too general.
"api" would be fine.